src/FacadeAccountCreation/FacadeAccountCreation.API/Controllers/OrganisationsController.cs

```diff
@@ -107,4 +108,41 @@ public async Task<IActionResult> GetNationIdByOrganisationId([FromQuery] string
 
         return response == null ? NotFound() : Ok(response);
     }
+
+    [HttpPost]
+    [Route("create-and-add-subsidiary")]
+    [Consumes("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    public async Task<ActionResult> CreateAndAddSubsidiary(LinkOrganisationModel linkOrganisationModel)
+    {
+        linkOrganisationModel.UserId = User.UserId();
+        var response = await _organisationService.CreateAndAddSubsidiaryAsync(linkOrganisationModel);
+
+        if (response == null)
+        {
+            return Problem("Failed to create and add organisation", statusCode: StatusCodes.Status500InternalServerError);
+        }
+
+        return Ok(response);
+    }
+
+    [HttpPost]
+    [Route("add-subsidiary")]
+    [Consumes("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    public async Task<ActionResult> AddSubsidiary(SubsidiaryAddModel subsidiaryAddModel)
+    {
+        subsidiaryAddModel.UserId = User.UserId();
+
+        var response = await _organisationService.AddSubsidiaryAsync(subsidiaryAddModel);
+
+        if (response == null)
+        {
+            return Problem("Failed to create and add organisation", statusCode: StatusCodes.Status500InternalServerError);
```

Comment:
Think error message is incorrect. It is same error as CreateAndAddSubsidiary method